src/GlobalStyles.js

```diff
@@ -42,7 +42,7 @@ body {
 }
 
 main {
-    margin: 8px 8px 68px 8px;
+    margin: 8px 12px 68px 12px;
```

Comment:
I think you could shorten it with the shorthand `margin: 8px 12px 68px;` if I am not mistaken